### PR TITLE
New build system for assets

### PR DESCRIPTION
**Pull request by romalytvynenko**
Fixes https://aventura.atlassian.net/browse/EDDBK-104

### CODE REVIEW

File: config/assets_urls_map.php [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
 
 return [
-    'require' => 'https://cdnjs.cloudflare.com/ajax/libs/require.js/2.3.5/require.js',
-
-    'wizard_ui/dist/app.min.js' => 'https://unpkg.com/@rebelcode/booking-wizard@0.1.5/dist/js/app.min',
-    'wizard_ui/dist/main.js'  => plugins_url(RC_BOOKINGS_FRONT_UI_MODULE_RELATIVE_DIR . '/dist/main.js', EDDBK_FILE),
-
-    'wizard_ui/dist/wp-bookings-front-ui.css' => plugins_url(RC_BOOKINGS_FRONT_UI_MODULE_RELATIVE_DIR . '/dist/wp-bookings-front-ui.css', EDDBK_FILE),
+    'wizard_ui/dist/app.min.js' => plugins_url(
+        RC_BOOKINGS_FRONT_UI_MODULE_RELATIVE_DIR . '/dist/' . RC_BOOKINGS_FRONT_UI_APP_VERSION . '/app.min.js',
```
Comment (mecha) [inline config/assets_urls_map.php:5]: These are the built bundles?

Comment (romalytvynenko) [reply config/assets_urls_map.php:5]: Yes.

Comment (mecha) [reply config/assets_urls_map.php:5]: Ok :+1:

**Comment by mecha** [comment]
From what I can see, this module no longer uses webpack, bower, etc, and assumes that something else will provide the bundles, correct?

**Comment by romalytvynenko** [comment]
@mecha, yes, correct. 

So js and css assets are now installed as a dependency of this module. After this, on `build` command, this bundles will be copied to `dist` folder and shipped with this module.

**Comment by mecha** [comment]
Nice. That sounds a lot better than before. The module is now a pure consumer of its JS and CSS dependencies and serves as an enqueuer. Approving.

**Review by mecha — APPROVED** [review]

**Pull request merged**